src/app/App.js

```diff
@@ -1,13 +1,26 @@
-import { Display, Log} from "../component/Add";
-import Navbar from "../component/Nav";
-
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { Login } from "../component/Login";
+import Header from "../component/Header";
+import Home from "../component/Home";
+import About from "../component/About";
+import Contact from "../component/Contact";
+import SignUp from "../component/Signup";
+import "./App.css";
 
 function App() {
   return (
     <>
-      <Navbar title="Student Management" />
-      <Display/> 
-      <Log/>
+      <BrowserRouter>
+        <Header />
+        <Routes>
+          <Route path="/" element={<Home />} />
```

Comment:
`<Route path="/for-sale" element={<Properties />} />` or `<Route path="/for-sale" Component={Properties} />`